src/components/Menu/keys.js

```diff
@@ -0,0 +1,24 @@
+const isMac = (navigator.userAgent.includes('Mac'))
```

Comment:
I would recommend to not use the `navigator.userAgent` which can be easily spoofed, but the more reliable `navigator.platform`.
So something like:
```js
/(Mac|iPhone|iPod|iPad)/i.test(navigator.platform)
```

Reply:
Unfortunately `platform` is deprecated 😥
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/platform

Reply:
Oh I did not know, guess I will have to rewrite some code.
I guess the future way to go is using the `userAgentData`, as at least [chrome plans migrate](https://developer.chrome.com/docs/privacy-sandbox/user-agent/) from the user agent string to the user agent client hints.

Of cause not every browser has implemented the client hints by now...
Meaning until Firefox and Safari implemented them a fallback like this must be used:

```js
const platform = navigator?.userAgentData?.platform || navigator?.platform || 'unknown'
```

Reply:
This is a good idea, unfortunately I don't have a Mac to proper test it.
Can you open an issue for tracking this in the future?

Reply:
> This is a good idea, unfortunately I don't have a Mac to proper test it. Can you open an issue for tracking this in the future?

Done, see #2770 :)